src/SessionList.ts

```diff
@@ -284,6 +281,10 @@ export class PersistentSessionList extends ASessionList {
       const myworkspaces = data.filter((d) => d.creator === me);
       const otherworkspaces = data.filter((d) => d.creator !== me);
 
+      const publicTitle = i18n.t('tdp:core.SessionList.status') as string;
+      const privateTitle = i18n.t('tdp:core.SessionList.status', {context: 'private'}) as string
```

Comment:
```suggestion
      const privateTitle = i18n.t('tdp:core.SessionList.status', {context: 'private'}) as string;
```

Is `as string` necessary? Because in the next line it does not seem to be necessary. Please revise accordingly and make them equal.

Reply:
In this case:

![error](https://user-images.githubusercontent.com/51322092/70032608-0f18d580-15ae-11ea-8f9a-a38e25cf821a.png)

You get this error:
`Argument of type '(d: IProvenanceGraphDataDescription) => string | object' is not assignable to parameter of type '(datum: IProvenanceGraphDataDescription, index: number, outerIndex: number) => Primitive'.
  Type 'string | object' is not assignable to type 'Primitive'.
    Type 'object' is not assignable to type 'Primitive'.
      Type 'object' is not assignable to type 'false'.`

And that gets resolved by adding the `as string` assertion or by assigning it to a variable.

![error2](https://user-images.githubusercontent.com/51322092/70032805-7c2c6b00-15ae-11ea-870b-9c40b02dd445.png)

I have encountered this error though only with `.attr` method.